client/components/forms/form-phone-media-input/index.jsx

```diff
@@ -30,6 +34,8 @@ export default class extends React.Component {
 					<PhoneInput
 						{ ...omit( this.props, [ 'className', 'countryCode' ] ) }
 						ref="input"
+						// eslint-disable-next-line react/jsx-no-bind
+						onRef={ ref => this.phoneInput = ref }
```

Comment:
You're setting a class var every time we mount the phone input? Right?

I believe this could also be defined as:
```
setPhoneInput = ref => this.phoneInput = ref
```

Reply:
So, this is an interesting question. I thought the same thing. The React documentation seems want you to use fat arrow functions for `ref` (https://facebook.github.io/react/docs/refs-and-the-dom.html)

But, yes, I could define the function and pass that in to onRef. Will update.

Reply:
I would keep it as it is - it's more in line with the react docs

Reply:
Oops. I updated before I read this. I'll see if I can figure out why the react docs use the anonymous function in render in this case.